<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>457</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow
{
background-color:#ffffff
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea{
border: 3px solid rgb(0, 170, 255);
background-color: #FFFFFF;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>423</width>
         <height>344</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelResult">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;华文楷体&quot;;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_copy">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font-weight:bold;
background-color: #FFFFFF;
color: rgb(0, 170, 255);
border-radius: 5px;
border: 2px groove rgb(0, 170, 255);
}
QPushButton:pressed{
font-weight:bold;
background-color: rgb(0, 170, 255);
color: #FFFFFF;
border-radius: 5px;
border: 2px groove rgb(0, 170, 255);
}

</string>
      </property>
      <property name="text">
       <string>复制到剪贴板</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>447</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="actionGrab"/>
    <addaction name="actionOCR"/>
    <addaction name="actionTrans"/>
    <addaction name="separator"/>
    <addaction name="actionFloat"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionFeedback"/>
    <addaction name="actionDonate"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionFeedback">
   <property name="text">
    <string>反馈</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>捐赠</string>
   </property>
   <property name="toolTip">
    <string>捐赠</string>
   </property>
  </action>
  <action name="actionGrab">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>框选截图</string>
   </property>
  </action>
  <action name="actionOCR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>框选OCR识别</string>
   </property>
  </action>
  <action name="actionTrans">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>框选翻译</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出软件</string>
   </property>
   <property name="toolTip">
    <string>退出软件</string>
   </property>
  </action>
  <action name="actionFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>悬浮窗模式</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
